core/storage/trie/impl/polkadot_codec.cpp

```diff
@@ -170,7 +170,7 @@ namespace kagome::storage::trie {
 
     if (node.getTrieType() == PolkadotNode::Type::BranchWithValue) {
       // scale encoded value
-      OUTCOME_TRY(encNodeValue, scale::encode(node.value));
+      OUTCOME_TRY(encNodeValue, scale::encode(node.value.get()));
```

Comment:
maybe we should add check if value is not none? Return error if it is empty

Reply:
Here it is unnecessary
Function `getTrieType` check it